service/service.go

```diff
@@ -127,3 +127,16 @@ func (s *Service) QueryClient() *panacea.QueryClient {
 func (s *Service) Ipfs() *ipfs.Ipfs {
 	return s.ipfs
 }
+
+func (s *Service) BroadcastTx(grpcClient *panacea.GrpcClient, txBytes []byte) (int64, string, error) {
+	resp, err := grpcClient.BroadcastTx(txBytes)
+	if err != nil {
+		return 0, "", fmt.Errorf("broadcast transaction failed. txBytes(%v)", txBytes)
+	}
```

Comment:
You don't need to pass the `grpcClient` since `Service` already has grpc client.

```suggestion
func (s *Service) BroadcastTx(txBytes []byte) (int64, string, error) {
	resp, err := s.GRPCClient().BroadcastTx(txBytes)
	if err != nil {
		return 0, "", fmt.Errorf("broadcast transaction failed. txBytes(%v)", txBytes)
	}
```

Reply:
Thanks! I fixed it.